Assets/MixedRealityToolkit/_Core/Inspectors/MixedRealityToolkitInspector.cs

```diff
@@ -53,7 +53,7 @@ public override void OnInspectorGUI()
                                                                   "Would you like to create one now?", "OK", "Later"))
                     {
                         ScriptableObject profile = CreateInstance(nameof(MixedRealityToolkitConfigurationProfile));
-                        profile.CreateAsset();
+                        profile.CreateAsset("Assets/CustomProfiles");
```

Comment:
Minor naming thing. Maybe call it something like "Assets/MixedRealityToolkit-generated/CustomProfiles"
to be aware where it belongs to in a bigger folder structure and it's generated?

Reply:
```suggestion
                        profile.CreateAsset("Assets/MixedRealityToolkit-generated/CustomProfiles");
```